tests/integration/conftest.py

```diff
@@ -826,7 +889,7 @@ def add_test_case(self,
             duration (int): The desired duration of the test in seconds. (Automatically generated if value is None)
             message (str): The desired message for the test which is only used for 'skipped', 'failure', 'error'
                 test states. (Automatically generated if value is None)
-            jira_tickets (list(str)): A list of Jira ticket IDs to associated with the test case.
+            jira_tickets ([str]): A list of Jira ticket IDs to associated with the test case.
```

Comment:
The JetBrains IDE does not recognize "list(*)" as a valid docstring data type. Fixed.